### PR TITLE
navbar

### PR DESCRIPTION
**Pull request by harshalini**
Added navigation component in component library.
The navigation/navbar contains -
1) site name
2) Search bar
3) Tabs

Preview ->
https://lucent-fairy-6f726a.netlify.app/documentation/documentation.html#navbar-div

### CODE REVIEW

**Review by munjalSaksham71 — APPROVED** [review]
Code looks good! Just a suggestion - please add a border around navbar to highlight the preview.

File: component.css [hunk 1]
```diff
@@ -11,4 +11,5 @@
 @import url(./components/modal/modal.css);
 @import url(./components/sliders/sliders.css);
 @import url(./components//grids/grid.css);
-@import url(./components/snackbar/snackbar.css);
\ No newline at end of file
+@import url(./components/snackbar/snackbar.css);
+@import url(./components/navbar/navbar.css);
```
Comment (ShreyashMahajan) [inline component.css:15]: liked that you have made separate css file for each component

**Review by ShreyashMahajan — APPROVED** [review]
The code is very understandable.
Naming is also very good.
Good to merge

**Review by Omkar-Chaskar — APPROVED** [review]
PR looks good to merge.
well structured , easy to understand and clean code.
Nice UI
keep it up.

**Pull request merged**